### PR TITLE
analytics: fix printing of opt-in status

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc,

Please review the following commits I made in branch nicks/cmd:

75ddb7b3d9884971465f792f1f921005069b4616 (2018-08-03 14:54:38 -0400)
analytics: fix printing of opt-in status

### CODE REVIEW

**Comment by nicks** [comment]
before this change, it would print `current collection strategy: 2`, which was...not great

File: pkg/analytics/cli.go [hunk 1]
```diff
@@ -11,12 +11,12 @@ import (
...
 
 func analyticsStatus(_ *cobra.Command, args []string) error {
-	//fmt.Println("[ for more info: https://windmill.engineering/analytics ]")
 	choice, err := OptStatus()
 	if err != nil {
 		return err
 	}
-	fmt.Printf("current collection strategy: %v\n", choice)
+
+	fmt.Printf("pets analytics status: %s\n", choice)
```
Comment (maiamcc) [inline pkg/analytics/cli.go]: this isn't pets-specific code :P

**Comment by nicks** [comment]
oooooooops :grimacing:

**Comment by nicks** [comment]
changed it to just 'analytics status'

**Pull request merged**